config/prod/supervisord.conf

```diff
@@ -0,0 +1,19 @@
+[supervisord]
+; Don't want supervisord to be daemonised, as otherwise when running under
+; Docker it will assume that the command has completed, whereas we want
+; supervisord to stay up as long as the application is running
+nodaemon=true
+
+
+[program:core_service]
+command=/usr/local/bin/gunicorn task_service.wsgi:application -c /code/config/prod/gunicorn.conf
+directory=/code
+; User account with minimal privileges
+user=nobody
```

Comment:
Does the _nobody_ user need to be created in the Dockerfile (something like: `RUN useradd nobody`), or is that not needed?